polar2grid/compare.py

```diff
@@ -110,19 +112,33 @@ def plot_array(array1, array2, cmap="viridis", vmin=None, vmax=None, **kwargs):
     if vmin is None:
         vmin = min(np.nanmin(array1), np.nanmin(array2))
         vmax = max(np.nanmax(array1), np.nanmax(array2))
-    fig, (ax1, ax2) = plt.subplots(1, 2)
-    ax1.imshow(array1, cmap=cmap, vmin=vmin, vmax=vmax)
-    ax2.imshow(array2, cmap=cmap, vmin=vmin, vmax=vmax)
+
+    fig, (ax1, ax2) = plt.subplots(2, 2)
+    array3 = (array1 - array2)
+    q=[0,0.25,0.5, 0.75,1.0]
+    array3_quantiles = np.nanquantile(array3, q)
+    fig.suptitle(array3_quantiles)
+
+    ax1[0].imshow(array1, cmap=cmap, vmin=vmin, vmax=vmax)
+    ax1[1].imshow(array2, cmap=cmap, vmin=vmin, vmax=vmax)
+
+    
+    diff_max = max(np.nanmax(array3),np.absolute(np.nanmin(array3)))
+    img3 = ax2[0].imshow(array3, cmap='RdBu', vmin=-diff_max, vmax=diff_max)
+    fig.colorbar(img3, ax=ax2[0])
+    fig.delaxes(ax2[1])
```

Comment:
Is this specific to the testing you are doing or should this be the new way these plots are done?

Reply:
I prefer to see a few numbers and a difference plot, but if it causes other issues, then this change can be rejected.

Reply:
No problem. Appreciated addition. I just wanted to make sure this wasn't left over from debugging. Could you remove the extra blank line above `diff_max = `?